charters/DataBrowser/charter.md

```diff
@@ -0,0 +1,66 @@
+# Data Browser
+
+
+## Description
+The Data Browser (available via the Data Portal 'Explore' section) is an application that provides a faceted search capability allowing users to search and download subsets of cellular resolution data hosted by the Data Coordination Platform (DCP).
+
+## Definitions
+*DCP* The Data Coordination Platform is the name given to the entire system used to ingest, validate, store, analyze, and make available the data in the Human Cell Atlas project.
+
+*Data Portal* The landing site for the Human Cell Atlas (HCA) DCP.
+
+*Analysis Applications* Third party portals listed in the 'Analyze' section of the Data Portal UI.
+
+## Objectives
+The objective of the Data Browser is to provide an easy-to-use User Interface (UI) and RESTful API allowing users to search, download and analyze data stored in the DCP at petabyte scale on public clouds.
+
+## In-scope
+
+### Core capabilities
+* Create indexes to access files in the Data Store by using public Data Store Service (DSS) APIs to read the metadata.
+* Support multiple metadata schemas via a shared schema access library.
+* Provide multiple search views of the DCP data such as by Projects, Files and Specimens.
+* Support for specifying multiple search criteria ('facets') through the UI.
+* Provide for download of selected files from the Data Store via the UI and/or HCA DCP Command Line Interface (CLI).
```

Comment:
When you use the word selected here do you mean user selected or that the browser will only enable download of a subset of files?

Reply:
Changed to "user-selected".